app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -355,6 +338,42 @@ class GithubWebhookSubscription extends SubscriptionHandler {
     await gitHubClient.pullRequests.createReview(slug, review);
   }
 
+  Future<void> _processPullRequestClosed(PullRequestEvent pullRequestEvent) async {
+    final PullRequest pr = pullRequestEvent.pullRequest!;
+
+    // Cancel any outstanding presubmit jobs. They are useless after the PR is
+    // closed. If the PR was merged, then the post-submit results will determine
+    // what needs to be done about this PR (maybe it lands, or maybe it will be
+    // reverted). If the PR was just closed and abandoned, well, that means we
+    // don't care about it any more.
+    await scheduler.cancelPreSubmitTargets(
+      pullRequest: pr,
+      reason: (!pr.merged!) ? 'Pull request closed' : 'Pull request merged',
+    );
+
+    if (pr.merged!) {
+      log.fine('Pull request ${pr.number} was closed and merged.');
+
+      // To avoid polluting the repo with temporary revert branches, delete the
```

Comment:
You might want to note why we're doing this here wrt to other actors (bots vs github vs merge queue).

Reply:
Done.